### ISSUE TITLE
with prior preservation always runs as "no" even though i set it to "yes"

### ISSUE THREAD

**Issue by dalmackay** [TheLastBen/fast-stable-diffusion#246]
mentioned this before but got no help. there are NO ERRORS. prior preservation is always set to YES, but the code always runs the "if prior preservation=NO" section of code. even though it is set to YES, it runs as NO. with no errors.

**Comment by TheLastBen** [comment]
did you run the setting up cells ?

**Comment by dalmackay** [comment]
yes. i havent done the FAST method, still trying the class image method. i've made about 15 models (with great results) but every single one ran with preservation off when it was always set to yes

**Comment by TheLastBen** [comment]
you must know that if there is already a folder of class images, it won't generate them again, it will directly start the training

**Comment by dalmackay** [comment]
yeah, i've tried generating them and ive tried uploading them. i guess i just dont know what prior preservation even is. just bugs me that it never runs as "yes". i feel like im missing out with it running as no. but my results are still good nonetheless

**Comment by TheLastBen** [comment]
try the new method, it's way better

**Comment by dalmackay** [comment]
yeah me and some others have been in the SD discord trying your new method as you update it. works very well for the cartoon characters i tried (1300 steps, 28 images, works with euler_a just as good as euler btw). thank you for making this so much easier than the other repos. idk why anyone uses the others at this point.

**Comment by TheLastBen** [comment]
they use wrong filenames for the pictures, that's why they get bad results, here is an example of training on 30 pics with 300 steps, 5 minutes

![download (1)](https://user-images.githubusercontent.com/110583491/198029973-fb058481-35b5-471d-be3c-b4c4bfe7cff8.png)

**Without training :**
![download (2)](https://user-images.githubusercontent.com/110583491/198030162-665fc3d0-e15a-40f3-8331-e802d3c948e7.png)

**Comment by dalmackay** [comment]
i've seen you suggesting anywhere from 300 to 1500 steps per model. time isnt an issue for me as its already so fast, which numbers do you have the best results with?
